src/parser/core/modules/WellPrepared.tsx

```diff
@@ -0,0 +1,73 @@
+import {Trans} from '@lingui/react'
+import {StatusLink} from 'components/ui/DbLink'
+import {StatusRoot} from 'data/STATUSES'
+import {Event, Events} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {filter} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import Checklist, {Requirement, Rule} from 'parser/core/modules/Checklist'
+import {Data} from 'parser/core/modules/Data'
+import React from 'react'
+
+// Give a use time of 5m10s between each potion use
+const TIME_BETWEEN_POTIONS_IN_MS = 310000
+
+export class WellPrepared extends Analyser {
+	static override handle = 'wellprepared'
+	static override debug = true
+
+	@dependency protected checklist!: Checklist
+	@dependency protected data!: Data
+
+	private wellFed: StatusRoot['WELL_FED'] = this.data.statuses.WELL_FED
+	private medicated: StatusRoot['MEDICATED'] = this.data.statuses.MEDICATED
+	private isWellFed: boolean = false
+	private potionUses: number = 0
+
+	override initialise() {
+		this.addEventHook(filter<Event>().type('actorUpdate').actor(this.parser.actor.id), this.onPull)
+		this.addEventHook(filter<Event>().type('statusApply').source(this.parser.actor.id)
+			.status(this.medicated.id), this.onPotionUse)
+		this.addEventHook('complete', this.onComplete)
+	}
+
+	private onComplete() {
+		this.debug(this.isWellFed)
+		this.debug(this.potionUses)
+		this.debug(this.parser.pull.duration)
+		this.debug(this.parser.pull.duration / TIME_BETWEEN_POTIONS_IN_MS)
+		this.debug(Math.floor(this.parser.pull.duration / TIME_BETWEEN_POTIONS_IN_MS))
```

Comment:
feel free to leave debug calls in, but it'd be nice to format this up as a single line entry with labels so a reader can understand what's going on